src/tools/dev/scripts/bv_support/bv_mili.sh

```diff
@@ -974,12 +854,6 @@ function apply_mili_patch
         if [[ $? != 0 ]] ; then
             return 1
         fi
-        if [[ "${MILI_FILE}" == "mili_15_1.tar.gz" ]]; then
-            apply_mili_151_darwin_patch4
-            if [[ $? != 0 ]] ; then
-                return 1
-            fi
-        fi
     fi
```

Comment:
Is 'apply_mili_151_darwin_patch4' no longer needed? If not, it should be removed from the file as well.